src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -111,13 +116,14 @@ protected void stopCustomBrowser(WebDriver customDriver) {
       try {
         customDriver.quit();
       } catch (Error e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Error. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)